src/permissions.ts

```diff
@@ -64,9 +64,11 @@ export default class Permissions {
   has(origin: Origin, paths: Array<string> = []): boolean {
     if (origin === SELF_ORIGIN) return true
     const currentPaths = this.get(origin)
-    return paths.reduce((acc: boolean, path: string) => {
-      return acc && currentPaths.includes(path)
-    }, Boolean(currentPaths))
+    return currentPaths == null
+      ? false
+      : paths.reduce((acc: boolean, path: string) => {
+          return acc && currentPaths.includes(path)
+        }, true)
```

Comment:
Not sure I follow why this is needed? If `Boolean(currentPaths)` doesn't work, can't we just do `Boolean(currentPaths == null)`?

Reply:
It works at runtime but at compile time TS was complaining about `currentPaths.includes` saying `currentPaths` could be `null`. In practice it's not true but the TS compiler is not smart enough to infer this.

Reply:
Can't we just do `currentPaths?.includes`?

Reply:
Yes we could do `!!currentPaths?.includes(path)` as `currentPaths?.includes` could return `null`, but I found it more readable this way. I can change it to `!!currentPaths?.includes(path)` though if you prefer?

Reply:
yeah, I think that's clearer than the turnary. (or `Boolean(currentPaths?.includes(path))`?)